src/scales/scale.time.js

```diff
@@ -237,9 +317,9 @@ function generate(scale, min, max, capacity) {
 	const options = scale.options;
 	const timeOpts = options.time;
 	const minor = timeOpts.unit || determineUnitForAutoTicks(timeOpts.minUnit, min, max, capacity);
-	const stepSize = resolve([timeOpts.stepSize, timeOpts.unitStepSize, 1]);
+	const stepSize = valueOrDefault(timeOpts.stepSize, 1);
```

Comment:
Removal of `unitStepSize` should be documented

Reply:
done